lib/cf_obs_binary_builder/obs_package.rb

```diff
@@ -1,7 +1,13 @@
+require "nokogiri"
 class CfObsBinaryBuilder::ObsPackage
   attr_reader :name, :obs_project
 
   ARTIFACT_EXTENSION_REGEXP=".tgz|.tar.gz|.tbz|tar.bz|.zip|.phar"
+  STACK_TO_OBS_REPOSITORY = {
+    "sle12" => "SLE_12_SP3",
+    "sle15" => "SLE_15",
```

Comment:
The stack is called `cfsle15fs`

Reply:
done

Reply:
might be relevang also for: https://github.com/SUSE/cf-obs-binary-builder/commit/69f75a171b6876c3559c964a500e8f255a7aa5d7#diff-c6acfca63885a53f914d8ee2530e6680R54  and https://github.com/SUSE/cf-obs-binary-builder/commit/f21e604b1af2aa29eceb39099710e76a89dfc3f1